README.md

```diff
@@ -265,7 +265,7 @@ You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
 
 Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
+distributed under the License is distributed on an "AS IS" BASIC,
```

Comment:
This is inccorrect. You cannot just go ahead an change LICENSE text, otherwise it it not anymore that license. The original is here:
http://www.apache.org/licenses/LICENSE-2.0